marketplacetests/firefox_accounts/app.py

```diff
@@ -50,5 +71,20 @@ def tap_next(self):
 
     def tap_sign_in(self):
         self.keyboard.dismiss()
+        self.change_to_correct_login_frame()
```

Comment:
This is the second time this is being called (it's also called in `__init__` at https://github.com/mozilla/marketplace-tests-gaia/pull/70/files#diff-4748f41d7933badeadce5c5003aaf2acR28), and it includes a test to check which frame to switch to. I suggest we store the information about which frame to switch to the first time this is called, and then the next time all we'll have to do is switch to that frame.